client/components/Landing/Footer.js

```diff
@@ -0,0 +1,58 @@
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+
+function Copyright() {
+    return (
+      <Typography variant="body2" sx={{
+          textAlign: "left",
+          fontWeight: "bold",
+          color: "#755CDE",
```

Comment:
get the purple color sa theme lang

Reply:
like this `<Typography sx={{color:"primary.main"}}>` or `<Typography color="primary">`